app/src/androidTest/java/nl/eduvpn/app/ui_test/ProviderSearchTest.kt

```diff
@@ -42,14 +42,16 @@ import org.junit.runner.RunWith
  */
 @RunWith(AndroidJUnit4::class)
 @LargeTest
+@Ignore("Sometimes fails in CI")
```

Comment:
Same as above, if we start ignoring the tests, then they are of no use.
This one is also pretty simple compared to the Chrome one, so I don't understand why it couldn't be fixed